### ISSUE TITLE
GW2 Target needs to be reset every new Launcher intance.

### ISSUE THREAD

**Issue by Wookie808** [Healix/Gw2Launcher#171]
A GW2 update a while back caused my Launcher to lose the GW2 target every time I close Launcher. So for the past while I have to reset the GW2 target when I start the Launcher. Is this a known issue and is there a fix? TIA

**Comment by Healix** [comment]
Do you mean the "Gw2.exe location" is reset every time you start Gw2Launcher?

**Comment by Wookie808** [comment]
Yeah, so if I kill Launcher and then restart it... and then try to launch GW2 using the launcher... it shows "failed" in red letters. But if I go into Settings -> GW2 -> and then reselect the GW2 location (even though it is already showing the correct location in the field) it will launch normally.

**Comment by Healix** [comment]
You can hover over the failed account to see the reason. What you're experiencing is likely the "object reference not set" bug in the release version. Changing the path of GW2 isn't actually fixing it, but rather simply opening the settings and clicking OK does.

To avoid the problem, right click any account > Settings > Guild Wars 2 > Launch options. For the arguments, add anything, such as -dx11 if you want DX11 to be the default, -maploadinfo etc. The bug occurs when the arguments are blank and you're using -dx11 or -dx9 on the account.

To fix the problem, use the [beta version](https://github.com/Healix/Gw2Launcher/tree/master/Gw2Launcher/bin64/Beta).

**Comment by Wookie808** [comment]
Hmmm… haven’t tried that yet but I do know that I have -dx11 in the
arguments but it never starts dx11 in any of my instances. I always have to
start it manually.

**Comment by Healix** [comment]
You start DX11 manually? If you mean the option isn't checked in-game, note that the -dx11 and -dx9 options are overrides (it doesn't matter if it's checked in-game). 

If you want to confirm the use of DX11:
Right click any account > Tools > Guild Wars 2 > Verify usage of DX11

**Comment by Wookie808** [comment]
Okay thank you.  I’ll try everything you’ve told me and get back to you.
Thank you again.

**Comment by Wookie808** [comment]
Well, I don't know why... but it's working now. Thank you for all your
help. Greatly appreciated.
